src/components/SQForm/SQFormMultiSelect.js

```diff
@@ -137,7 +139,7 @@ function SQFormMultiSelect({
           input={<Input disabled={isDisabled} name={name} />}
           value={field.value}
           onBlur={handleBlur}
-          onChange={handleMultiSelectChange}
+          onChange={handleMultiSelectChange && handleChange}
```

Comment:
With this change `handleMultiSelectChange` will never get called, right? Maybe I'm missing something.

Reply:
If we add a custom `onChange` prop to this component we want the original `handleMultiSelectChange` method to be called. Then if an `onChange` method is passed to the component, call it after the `handleMultiSelectChange` method has executed.

This component is imperatively calling setFieldValue already so I believe we wouldn't use the `handleChange` help method like we would in fields that are provided a single value.

I think what we need to do here is after `setFieldValue(name, values);` is called, add a new line with `onChange && onChange(value);`

Reply:
Good call.  Overlooked that by trying to use the useForm hook.  @SeanGroff and @20BBrown14 I have updated the code.
![image](https://user-images.githubusercontent.com/20329437/104349497-9b433f00-54c8-11eb-84b6-ca61eece3078.png)